### PR TITLE
Fix Motionblinds brand name consistency in virtual integrations

### PR DESCRIPTION
**Pull request by LennP**
## Proposed change

Corrects the name of **Motionblinds** and **Smartblinds** for all virtual integrations of motion_blinds.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by LennP** [comment]
In the other PR I hadn't corrected the names in the virtual integration pages @starkillerOG

**Comment by c0ffeeca7** [comment]
thank you for the catch, @LennP :+1: 
quick question: why change the single quote?

**Comment by LennP** [comment]
> quick question: why change the single quote?

That's what I was wondering too afterwards. I suppose it was some vscode autocomplete that changed them to double quotes.

**Comment by c0ffeeca7** [comment]
> > quick question: why change the single quote?
> 
> That's what I was wondering too afterwards. I suppose it was some vscode autocomplete that changed them to double quotes.

Oh, I see. I've had that happen to me before. Could you revert the change? The rest looks good to me.

**Comment by LennP** [comment]
> > > quick question: why change the single quote?
> > 
> > 
> > That's what I was wondering too afterwards. I suppose it was some vscode autocomplete that changed them to double quotes.
> 
> Oh, I see. I've had that happen to me before. Could you revert the change? The rest looks good to me.

Sure do you happen to know which extension forces these double quotes? Every time I try to save it changes back to double quotes haha

**Comment by c0ffeeca7** [comment]
> > > > quick question: why change the single quote?
> > > 
> > > 
> > > That's what I was wondering too afterwards. I suppose it was some vscode autocomplete that changed them to double quotes.
> > 
> > 
> > Oh, I see. I've had that happen to me before. Could you revert the change? The rest looks good to me.
> 
> Sure do you happen to know which extension forces these double quotes? Every time I try to save it changes back to double quotes haha

Not from the top of my head. Let me fix it from my end. Give me a minute.

File: source/_integrations/amp_motorization.markdown [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
   - Cover
 ha_domain: amp_motorization
 ha_integration_type: virtual
 ha_supporting_domain: motion_blinds
-ha_supporting_integration: Motion Blinds
+ha_supporting_integration: Motionblinds
 ha_release: 2020.12
 ha_codeowners:
-  - '@starkillerOG'
+  - "@starkillerOG"
```
Comment (c0ffeeca7) [inline source/_integrations/amp_motorization.markdown]:
```suggestion
  - '@starkillerOG'
```

**Comment by LennP** [comment]
> > > > > quick question: why change the single quote?
> > > > 
> > > > 
> > > > That's what I was wondering too afterwards. I suppose it was some vscode autocomplete that changed them to double quotes.
> > > 
> > > 
> > > Oh, I see. I've had that happen to me before. Could you revert the change? The rest looks good to me.
> > 
> > 
> > Sure do you happen to know which extension forces these double quotes? Every time I try to save it changes back to double quotes haha
> 
> Not from the top of my head. Let me fix it from my end. Give me a minute.

It looks like installing 'Markdown All in One' extension fixes it

File: source/_integrations/bliss_automation.markdown [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
   - Cover
 ha_domain: bliss_automation
 ha_integration_type: virtual
 ha_supporting_domain: motion_blinds
-ha_supporting_integration: Motion Blinds
+ha_supporting_integration: Motionblinds
 ha_release: 2020.12
 ha_codeowners:
-  - '@starkillerOG'
+  - "@starkillerOG"
```
Comment (c0ffeeca7) [inline source/_integrations/bliss_automation.markdown]:
```suggestion
  - '@starkillerOG'
```

File: source/_integrations/brel_home.markdown [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
   - Cover
 ha_domain: brel_home
 ha_integration_type: virtual
 ha_supporting_domain: motion_blinds
-ha_supporting_integration: Motion Blinds
+ha_supporting_integration: Motionblinds
 ha_release: 2020.12
 ha_codeowners:
-  - '@starkillerOG'
+  - "@starkillerOG"
```
Comment (c0ffeeca7) [inline source/_integrations/brel_home.markdown]:
```suggestion
  - '@starkillerOG'
```

File: source/_integrations/bloc_blinds.markdown [hunk 4]
```diff
@@ -1,15 +1,15 @@
...
   - Cover
 ha_domain: bloc_blinds
 ha_integration_type: virtual
 ha_supporting_domain: motion_blinds
-ha_supporting_integration: Motion Blinds
+ha_supporting_integration: Motionblinds
 ha_release: 2020.12
 ha_codeowners:
-  - '@starkillerOG'
+  - "@starkillerOG"
```
Comment (c0ffeeca7) [inline source/_integrations/bloc_blinds.markdown]:
```suggestion
  - '@starkillerOG'
```

File: source/_integrations/gaviota.markdown [hunk 5]
```diff
@@ -1,15 +1,15 @@
...
   - Cover
 ha_domain: gaviota
 ha_integration_type: virtual
 ha_supporting_domain: motion_blinds
-ha_supporting_integration: Motion Blinds
+ha_supporting_integration: Motionblinds
 ha_release: 2020.12
 ha_codeowners:
-  - '@starkillerOG'
+  - "@starkillerOG"
```
Comment (c0ffeeca7) [inline source/_integrations/gaviota.markdown]:
```suggestion
  - '@starkillerOG'
```

File: source/_integrations/havana_shade.markdown [hunk 6]
```diff
@@ -1,15 +1,15 @@
...
   - Cover
 ha_domain: havana_shade
 ha_integration_type: virtual
 ha_supporting_domain: motion_blinds
-ha_supporting_integration: Motion Blinds
+ha_supporting_integration: Motionblinds
 ha_release: 2020.12
 ha_codeowners:
-  - '@starkillerOG'
+  - "@starkillerOG"
```
Comment (c0ffeeca7) [inline source/_integrations/havana_shade.markdown]:
```suggestion
  - '@starkillerOG'
```

File: source/_integrations/hurrican_shutters_wholesale.markdown [hunk 7]
```diff
@@ -1,15 +1,15 @@
...
   - Cover
 ha_domain: hurrican_shutters_wholesale
 ha_integration_type: virtual
 ha_supporting_domain: motion_blinds
-ha_supporting_integration: Motion Blinds
+ha_supporting_integration: Motionblinds
 ha_release: 2020.12
 ha_codeowners:
-  - '@starkillerOG'
+  - "@starkillerOG"
```
Comment (c0ffeeca7) [inline source/_integrations/hurrican_shutters_wholesale.markdown]:
```suggestion
  - '@starkillerOG'
```

File: source/_integrations/inspired_shades.markdown [hunk 8]
```diff
@@ -1,15 +1,15 @@
...
   - Cover
 ha_domain: inspired_shades
 ha_integration_type: virtual
 ha_supporting_domain: motion_blinds
-ha_supporting_integration: Motion Blinds
+ha_supporting_integration: Motionblinds
 ha_release: 2020.12
 ha_codeowners:
-  - '@starkillerOG'
+  - "@starkillerOG"
```
Comment (c0ffeeca7) [inline source/_integrations/inspired_shades.markdown]:
```suggestion
  - '@starkillerOG'
```

File: source/_integrations/ismartwindow.markdown [hunk 9]
```diff
@@ -1,15 +1,15 @@
...
   - Cover
 ha_domain: ismartwindow
 ha_integration_type: virtual
 ha_supporting_domain: motion_blinds
-ha_supporting_integration: Motion Blinds
+ha_supporting_integration: Motionblinds
 ha_release: 2020.12
 ha_codeowners:
-  - '@starkillerOG'
+  - "@starkillerOG"
```
Comment (c0ffeeca7) [inline source/_integrations/ismartwindow.markdown]:
```suggestion
  - '@starkillerOG'
```

File: source/_integrations/martec.markdown [hunk 10]
```diff
@@ -1,15 +1,15 @@
...
   - Cover
 ha_domain: martec
 ha_integration_type: virtual
 ha_supporting_domain: motion_blinds
-ha_supporting_integration: Motion Blinds
+ha_supporting_integration: Motionblinds
 ha_release: 2020.12
 ha_codeowners:
-  - '@starkillerOG'
+  - "@starkillerOG"
```
Comment (c0ffeeca7) [inline source/_integrations/martec.markdown]:
```suggestion
  - '@starkillerOG'
```

File: source/_integrations/raven_rock_mfg.markdown [hunk 11]
```diff
@@ -1,15 +1,15 @@
...
   - Cover
 ha_domain: raven_rock_mfg
 ha_integration_type: virtual
 ha_supporting_domain: motion_blinds
-ha_supporting_integration: Motion Blinds
+ha_supporting_integration: Motionblinds
 ha_release: 2020.12
 ha_codeowners:
-  - '@starkillerOG'
+  - "@starkillerOG"
```
Comment (c0ffeeca7) [inline source/_integrations/raven_rock_mfg.markdown]:
```suggestion
  - '@starkillerOG'
```

File: source/_integrations/screenaway.markdown [hunk 12]
```diff
@@ -1,15 +1,15 @@
...
   - Cover
 ha_domain: screenaway
 ha_integration_type: virtual
 ha_supporting_domain: motion_blinds
-ha_supporting_integration: Motion Blinds
+ha_supporting_integration: Motionblinds
 ha_release: 2020.12
 ha_codeowners:
-  - '@starkillerOG'
+  - "@starkillerOG"
```
Comment (c0ffeeca7) [inline source/_integrations/screenaway.markdown]:
```suggestion
  - '@starkillerOG'
```

File: source/_integrations/smart_blinds.markdown [hunk 13]
```diff
@@ -1,15 +1,15 @@
...
   - Cover
 ha_domain: smart_blinds
 ha_integration_type: virtual
 ha_supporting_domain: motion_blinds
-ha_supporting_integration: Motion Blinds
+ha_supporting_integration: Motionblinds
 ha_release: 2020.12
 ha_codeowners:
-  - '@starkillerOG'
+  - "@starkillerOG"
```
Comment (c0ffeeca7) [inline source/_integrations/smart_blinds.markdown]:
```suggestion
  - '@starkillerOG'
```

File: source/_integrations/smart_home.markdown [hunk 14]
```diff
@@ -1,15 +1,15 @@
...
   - Cover
 ha_domain: smart_home
 ha_integration_type: virtual
 ha_supporting_domain: motion_blinds
-ha_supporting_integration: Motion Blinds
+ha_supporting_integration: Motionblinds
 ha_release: 2020.12
 ha_codeowners:
-  - '@starkillerOG'
+  - "@starkillerOG"
```
Comment (c0ffeeca7) [inline source/_integrations/smart_home.markdown]:
```suggestion
  - '@starkillerOG'
```

File: source/_integrations/uprise_smart_shades.markdown [hunk 15]
```diff
@@ -1,15 +1,15 @@
...
   - Cover
 ha_domain: uprise_smart_shades
 ha_integration_type: virtual
 ha_supporting_domain: motion_blinds
-ha_supporting_integration: Motion Blinds
+ha_supporting_integration: Motionblinds
 ha_release: 2020.12
 ha_codeowners:
-  - '@starkillerOG'
+  - "@starkillerOG"
```
Comment (c0ffeeca7) [inline source/_integrations/uprise_smart_shades.markdown]:
```suggestion
  - '@starkillerOG'
```

**Comment by LennP** [comment]
Thanks!

File: source/_integrations/diaz.markdown [hunk 16]
```diff
@@ -1,15 +1,15 @@
...
   - Cover
 ha_domain: diaz
 ha_integration_type: virtual
 ha_supporting_domain: motion_blinds
-ha_supporting_integration: Motion Blinds
+ha_supporting_integration: Motionblinds
 ha_release: 2020.12
 ha_codeowners:
-  - '@starkillerOG'
+  - "@starkillerOG"
```
Comment (c0ffeeca7) [inline source/_integrations/diaz.markdown]:
```suggestion
  - '@starkillerOG'
```

File: source/_integrations/dooya.markdown [hunk 17]
```diff
@@ -1,15 +1,15 @@
...
   - Cover
 ha_domain: dooya
 ha_integration_type: virtual
 ha_supporting_domain: motion_blinds
-ha_supporting_integration: Motion Blinds
+ha_supporting_integration: Motionblinds
 ha_release: 2020.12
 ha_codeowners:
-  - '@starkillerOG'
+  - "@starkillerOG"
```
Comment (c0ffeeca7) [inline source/_integrations/dooya.markdown]:
```suggestion
  - '@starkillerOG'
```

**Comment by c0ffeeca7** [comment]
> > > > > > quick question: why change the single quote?
> > > > > 
> > > > > 
> > > > > That's what I was wondering too afterwards. I suppose it was some vscode autocomplete that changed them to double quotes.
> > > > 
> > > > 
> > > > Oh, I see. I've had that happen to me before. Could you revert the change? The rest looks good to me.
> > > 
> > > 
> > > Sure do you happen to know which extension forces these double quotes? Every time I try to save it changes back to double quotes haha
> > 
> > 
> > Not from the top of my head. Let me fix it from my end. Give me a minute.
> 
> It looks like installing 'Markdown All in One' extension fixes it

ah, I learned something. Thank you!

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @LennP :+1:

**Comment by LennP** [comment]
> Thank you, @LennP 👍

Thank you too for the review!

**Pull request merged**